src/Slack/SlackService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta\Slack;
+
+use CL\Slack\Payload\PayloadInterface;
+use CL\Slack\Payload\PayloadResponseInterface;
+use CL\Slack\Transport\ApiClient;
+
+class SlackService
```

Comment:
:thinking: other classes in this namespace are not prefixed:

![image](https://user-images.githubusercontent.com/225704/39037717-97d0c360-4481-11e8-9d42-b68cbc9c92c5.png)

Reply:
That's true but I just made it coherent with Discord classes.